aiographite/graphite_escaping.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import urllib.parse
+
+"""
+	Graphite expects everything to be just ASCII to split/processing them, and then make directories based on metric name.
+	So any special name not allow to appear in directory/file name is not supported by Graphite.
+"""
+
+"""
+	Naming metrics schema:
+		<namespace>.<instrumented section>.<target (noun)>.<action (past tense verb)>
+"""
+
+"""
+	Limitation and explaination:
```

Comment:
is this copy pasted from the documentation? we should probably just link, in that case.

copy-paste can become out of date.